src/main/java/mcjty/theoneprobe/apiimpl/providers/DefaultProbeInfoProvider.java

```diff
@@ -188,10 +190,15 @@ private void showRF(IProbeInfo probeInfo, World world, BlockPos pos) {
             if (handler != null) {
                 addRFInfo(probeInfo, config, handler.getEnergyStored(), handler.getMaxEnergyStored());
             }
+        } else if (Loader.isModLoaded("tesla") && te != null && te.hasCapability(TeslaCapabilities.CAPABILITY_HOLDER, null)) {
```

Comment:
I've tested it. I'll post some screenshots in a couple minutes (starting everything)